db/migrate/20240624115536_add_indices_to_invoices.rb

```diff
@@ -0,0 +1,10 @@
+class AddIndicesToInvoices < ActiveRecord::Migration[7.0]
+  def change
+    add_index :invoices, :currency
+    add_index :invoices, :issuing_date
+    add_index :invoices, :invoice_type
+    add_index :invoices, :status
+    add_index :invoices, :payment_status
+    add_index :invoices, :payment_dispute_lost_at
```

Comment:
I'm not sure these index would be used often if we combine conditions, which I think would be pretty often 🤔

Reply:
I was expecting that postgres will use one of the indices or use Bitmap Index Scans to combine them. While having a combined index is more performant that option is not feasible for our query filters.

Reply:
I'm dropping all the new indices.